src/main/java/springbootquickstarter/entities/Student.java

```diff
@@ -75,20 +71,18 @@ public void setIdNumber(Long idNumber) {
 		this.idNumber = idNumber;
 	}
 
-	public Set <Course> getCourses() {
+	public Set<Course> getCourses() {
 		return courses;
 	}
 
-
 	@SuppressWarnings("unchecked")
 	public void setCourses(Long courseId, String string1, String string2, Long topicId) {
-		Set <Course> course = (Set <Course>) new Course(courseId, string1,string2, topicId);
+		Set<Course> course = (Set<Course>) new Course(courseId, string1, string2, topicId);
```

Comment:
why do it like this? if we have a list, then why not to add new element to it?

Reply:
Modified.